progress-reports/icon-bridge/August-2022-BTP-Progress-Update.md

```diff
@@ -0,0 +1,36 @@
+## Intro
+
+This is Icon Bridge by Web3Lab. This is progress report from 1-August-2022 to 31-August-2022
+
+## Summary
+
+Have created a CI/CD process based on Github action scripts.
+Has been analyzed project dependencies and updated  that  have had vulnerabilities.
+Have started a process of improving existing testing approaches and test coverage in the Golang project part.
+
+## Deliverables
+
+### Accomplished this cycle
+
+__Deliverables ready__
+
+| Name                                                                | Development State | Notes | Source / location                                                                                                                                                                                                                                                                                                                                    |
+|---------------------------------------------------------------------| ----------------- |-------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Create a release process for BTP project                            | Released |       | https://app.zenhub.com/workspaces/icon-bridge-625892ed86f6a3001152d8f8/issues/icon-project/icon-bridge/124                                                                                                                                                                                                                                           |
```

Comment:
Please link github issues instead of their associated zenhub link because github is open to the public but zenhub is not.